news/8313.hooks.rst

```diff
@@ -0,0 +1 @@
+Remove run-time hook for ``win32com``, as per discussion in issue:`8309`.
```

Comment:
It's actually
```suggestion
Remove run-time hook for ``win32com``, as per discussion in :issue:`8309`.
```
but it's easier if you just leave it and I'll mop it up at release time.